test/pet.test.js

```diff
@@ -0,0 +1,189 @@
+const Pet = require('../src/pet');
+
+let testPet;
+
+const deadMessage = "Your pet is no longer alive :("
+
+beforeEach(() => {
+    testPet = new Pet();
+  });
+
+describe('constructor', () => {
+    it('should return a new pet with a name', () => {
+        expect(new Pet()).toBeInstanceOf(Object);
```

Comment:
you could also separate this test into it's own block and it should return an object